projects/ngx-konva/src/lib/directives/ko-transform.directive.ts

```diff
@@ -12,6 +13,17 @@ import { KoNestable, KoNestableNode } from '../common/ko-nestable';
   }],
 })
 export class KoTransformDirective implements OnInit, OnDestroy {
+  @Input() set
+  koTransformEnabled(enable:boolean) {
```

Comment:
I'd put an Input called koTransform that will receive an object if enabled (empty if there are any defaults), a falsy value if disabled. With this solution we can have a unique directive koTransform both for enabling and setting the options.

Reply:
That was my initial idea, but unfortunately one of the Outputs is named koTransform so there's a clash there.

Reply:
Shut, you're right. Never mind then